tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/admin/OrgsSpec.scala

```diff
@@ -282,13 +282,6 @@ class OrgsSpec extends BaseSpec with EitherValuable {
       )
     }
 
-    "fail when wrong revision is provided" taggedAs OrgsTag in {
```

Comment:
why are you removing this test?

Reply:
I thought it was duplicate of the test from 193, but it's actually different. I'll add it back.